eng/jobs/finalize-publish.yml

```diff
@@ -0,0 +1,94 @@
+parameters:
+  Jobs: []
+jobs:
+- ${{ each job in parameters.Jobs }}: 
+  - job: ${{ job.displayName }}
+    displayName: ${{ job.displayName }}
+    # Run only if all build legs succeeded
+    condition: and(succeeded('Linux_x64_glibc'),
```

Comment:
Is this necessary? Isn't this the default for dependsOn?

Reply:
Private testing confirmed that dependsOn just waits for the execution of the dependent job but doesn't check the run result ( succeeded or failed ) . Thus, added this condition.

Reply:
Hmmm... that seems broken. 

https://docs.microsoft.com/en-us/azure/devops/pipelines/process/multiple-phases?tabs=yaml&view=vsts#conditions

> By default, a job runs if it does not depend on any other job, or if all of the jobs that it depends on have completed and succeeded. You can customize this behavior by forcing a job to run even if a previous job fails or by specifying a custom condition.